governance/process.md

```diff
@@ -1,144 +1,209 @@
+   you are interested in leading, comment on the issue and reach out in the
+   [#tag-security-triage](https://cloud-native.slack.com/archives/CJX8EBF1D)
+   channel to bring attention to it. Skip to the 3rd step.
 
-[Create an issue](https://github.com/cncf/tag-security/issues/new)
-that outlines the problem to be solved. Use the proposal template if you
-are interested in leading the work, or suggestion template if you would like
-someone else to lead the effort. If possible, include:
-
-    * The customer impact of the problem
-    * The scope of the work required
+1. **Raise an Issue:** [Create an
+   issue](https://github.com/cncf/tag-security/issues/new) that outlines the
+   problem to be solved. Use the proposal template if you are interested in
+   leading the work and _be sure to complete the project lead intent checkbox_,
+   or use the suggestion template if you would like someone else to lead the
+   effort. Try to complete the template as much as possible.
 
 1. **Ask the group for collaboration:** Rather than immediately beginning work
-on a solution, bring the issue up for discussion. The following guidance shows
-common steps, though communication often happens in different sequence. The
-key outcome is that there is opportunity for input across different channels,
-with thoughtfulness to accessibility across timezone and communication medium.
-We also encourage outreach outside of the group, when there are experts who
-might share insights (via invited presentation) or wish to get involved.
+   on a solution, bring the issue up for discussion in a Security TAG meeting.
+   The following guidance shows common steps, though communication often happens
+   in different sequence. The key outcome is that there is opportunity for input
+   across different channels, with thoughtfulness to accessibility across
+   timezone and communication medium. We also encourage outreach outside of the
+   group, when there are experts who might share insights (via invited
+   presentation) or wish to get involved.
 
    A) On slack, share the issue link and ask whether others are interested in
-   the problem and have feedback on your proposed solution or activity.
-
-   B) Choose an upcoming meeting where you or another group member who is
-   interested in working on the project is able to attend, then add the issue
-   to the meeting agenda: include a link and the name of the person who will
-   present the proposal in the "Planned Meeting" area of the
-   [meeting notes][https://github.com/cncf/tag-security#meeting-times].
-   Then at the meeting:
+   the problem, if so, let them know to join the planned meeting (see below)
+   where the issue will be introduced or where to stay tuned for more
+   information. provide the issue link and encourage any feedback on your
+   proposed solution or activity.
+
+   B) Choose an upcoming meeting in **_either region_** where you or another
```

Comment:
```suggestion
   B) Choose an upcoming meeting where you or another
```
let's not encode two regions!    We may someday split Europe/Africa from Americas